src/webots/vrml/WbProtoManager.cpp

```diff
@@ -137,6 +137,8 @@ WbProtoModel *WbProtoManager::findModel(const QString &modelName, const QString
       if (proto->name() == modelName && (proto->isImportable() || proto->isFromRootNodeConversion()))
         protoDeclaration = proto->url();
     }
+
+    mImportedFromSupervisor = false;
     // for supervisor imported nodes, there should always be a corresponding PROTO in the IMPORTABLE list
     assert(!mImportedFromSupervisor || !protoDeclaration.isEmpty());
```

Comment:
This assert is obviously always true because `mImportedFromSupervisor` is always false.

Reply:
Indeed, I had converted the PR back to draft but you were too fast.